README.md

```diff
@@ -1,3 +1,618 @@
+                <encoding>UTF-8</encoding>
+                <consoleOutput>true</consoleOutput>
+                <failsOnError>true</failsOnError>
+                <linkXRef>false</linkXRef>
+            </configuration>
+        </plugin>
+    </plugins>
+```
+Be aware of current versions.
+
+
+### Gradle
+Include [Checstyle Plugin](https://docs.gradle.org/current/userguide/checkstyle_plugin.html) in the head of `build.gradle` file.
+```
+plugins {
+    id 'checkstyle'
+}
+
+checkstyle {
+    toolVersion '8.42'
+    config project.resources.text.fromUri(new URI("https://raw.githubusercontent.com/pepperkit/checkstyle/v1.0.0-8.45/checkstyle.xml"))
+}
+```
+Be aware of current versions or any compitible versions.
+It executes in the build task automatically.
+
+## Versioning
+When this style guide is changed, its version increases, it follows semver logic. However, to highlight with which
+version of Checkstyle this style guide is compatible, the corresponding version of Checkstyle is stated in the version of 
+the style guide.
+
+For example, if the style guide has version `1.0.0` and it is compatible with Checkstyle version `9.0`, then the version
+of Checkstyle file will be `v1.0.0-9.0`.
+
+## Style Guide
+
+### Project Structure
+
+The project's structure has to conform to [Standard Directory Layout](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html).
+The examples are provided below.
+
+#### Apache Maven
+```text
+project-name
+├── pom.xml
+├── .gitignore
+├── LICENSE
+├── CHANGELOG.md
+├── README.md
+└── src
+    ├── docker
+    │   └── Dockerfile
+    ├── itest
+    │   ├── java
+    │   ├── kotlin
+    │   └── resources
+    ├── main
+    │   ├── java
+    │   ├── kotlin
+    │   └── resources
+    └── test
+        ├── java
+        ├── kotlin
+        └── resources
+```
+
+#### Gradle
+```text
+project-name
+├── build.gradle
+├── gradle.properties
+├── setting.gradle
+├── .gitignore
+├── LICENSE
+├── CHANGELOG.md
+├── README.md
+└── src
+    ├── docker
+    │   └── Dockerfile
+    ├── itest
+    │   ├── java
+    │   ├── kotlin
+    │   └── resources
+    ├── main
+    │   ├── java
+    │   ├── kotlin
+    │   └── resources
+    └── test
+        ├── java
+        ├── kotlin
+        └── resources
+```
+
+### Source Code File
+
+#### Encoding
+Files with the source code have to be in `UTF-8` encoding.
+
+#### Alignment
+* Use spaces (not tabs), one indent - 4 spaces
+* Lines should end with *LF* (Unix way), not CRLF (Windows way)
+* Line's length should not exceed 120 symbols (except import statements)
+* Source code file should end with an empty line
+
+#### Source Code Files Structure
+1. License
+2. Package statements
+3. _Empty line_
+4. Import statements
+5. _Empty line_
+6. One high-level class
+
+### License
+The license should be put in each source code file in a header.
+
+#### An example of full license text
+```java
+ /*
+ * Copyright (C) 2021 Arthur Aukhatov
```

Comment:
I really appreciate it)
But we should change it to **Pepper Kit**

Reply:
Ok :)

Reply:
done